pkgs/development/compilers/purescript/purescript/default.nix

```
@@ -24,10 +24,17 @@ in stdenv.mkDerivation rec {
   src =
     if stdenv.isDarwin
     then
-    fetchurl {
-      url = "https://github.com/${pname}/${pname}/releases/download/v${version}/macos.tar.gz";
-      sha256 = "1xxg79rlf7li9f73wdbwif1dyy4hnzpypy6wx4zbnvap53habq9f";
-    }
+      (if stdenv.system == "aarch64-darwin"
```

This should probably be

```suggestion
      (if stdenv.isAarch64
```